app/assets/javascripts/incident_reports/edit.coffee

```diff
@@ -54,17 +54,16 @@ deleteInjuredPassengerFields = (event) ->
   deleteFields '.pax-fields', '#supervisor_report_inj_pax_info'
 
 toggleReasonsForTesting = ->
-  reason = $('#supervisor_report_reason_test_completed').val()
-  switch reason
-    when 'Post-Accident'
-      $('.post-accident-info').slideDown()
-      $('.reasonable-suspicion-info').slideUp()
-    when 'Reasonable Suspicion'
-      $('.post-accident-info').slideUp()
-      $('.reasonable-suspicion-info').slideDown()
-    when ''
-      $('.post-accident-info').slideUp()
-      $('.reasonable-suspicion-info').slideUp()
+  reason = $('#supervisor_report_test_status').val()
+  if reason.search('Post Accident') == 0
```

Comment:
This JS really needs to be cleaned up. Perhaps we could add a function that returns the relevant information based on the input -- it could return "Post Accident", "Reasonable Suspicion", or blank, then we could go back to using the `switch` statement which is much clearer.
There could be better ways of abstracting some of this logic out because it's too much for just one function but that's a start.